File: forge/src/main/java/com/possible_triangle/forge/BourgeoisieForge.java
package com.possible_triangle.forge;

import com.possible_triangle.Bourgeoisie;
import me.shedaniel.architectury.platform.forge.EventBuses;
import net.minecraftforge.fml.common.Mod;
import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;

@Mod(Bourgeoisie.MOD_ID)
public class BourgeoisieForge {
    public BourgeoisieForge() {
        EventBuses.registerModEventBus(Bourgeoisie.MOD_ID, FMLJavaModLoadingContext.get().getModEventBus());
        Bourgeoisie.init();
    }
}

File: forge/src/main/java/com/possible_triangle/forge/BourgeoisieForge.kt
package com.possible_triangle.forge

import com.possible_triangle.Bourgeoisie
import me.shedaniel.architectury.platform.forge.EventBuses
import net.minecraftforge.fml.common.Mod
import thedarkcolour.kotlinforforge.forge.MOD_CONTEXT

@Mod(Bourgeoisie.MOD_ID)
class BourgeoisieForge {
    init {
        EventBuses.registerModEventBus(Bourgeoisie.MOD_ID, MOD_CONTEXT.getKEventBus())
        Bourgeoisie.init()
    }
}